### PR TITLE
feat: Vehicle journeys

### PR DESCRIPTION
**Pull request by alainajaffer**
Run retrievers again and rename tables

Use the local api url (change the test one to the aws one) in the site to see if wigan 1 is different to wigan 2 under Tfgm

### CODE REVIEW

File: packages/api/client.ts [hunk 1]
```diff
@@ -352,7 +352,12 @@ export const getServiceStops = async (dbClient: Kysely<Database>, input: Service
 
     const stops = await dbClient
         .selectFrom("services")
-        .innerJoin("service_journey_patterns", "service_journey_patterns.operatorServiceId", "services.id")
+        .innerJoin("vehicle_journeys", "vehicle_journeys.lineRef", "services.lineId")
+        .innerJoin(
+            "service_journey_patterns",
+            "service_journey_patterns.journeyPatternRef",
+            "vehicle_journeys.journeyPatternRef",
+        )
```
Comment (alvak1) [inline packages/api/client.ts]: Could we use below code which will allow us to remove the where in in line 419 on `service_journey_patterns.operatorServiceId`

`.innerjoin("service_journey_patterns", (join) => join.onRef("service_journey_patterns.journeyPatternRef","=","vehicle_journeys.journeyPatternRef").on("service_journey_patterns. operatorServiceId","=","services.id"))`

File: packages/ref-data-uploaders/txc-uploader/txc_processor.py [hunk 2]
```diff
@@ -541,6 +593,18 @@ def write_to_database(
                                 logger
                             )
 
+                for vehicle in vehicle_journeys:
```
Comment (alvak1) [inline packages/ref-data-uploaders/txc-uploader/txc_processor.py:595]: We are running insert query within a for loop. Could we run a batch insert to insert many vehicle journey rows?

File: packages/ref-data-uploaders/txc-uploader/txc_processor.py [hunk 3]
```diff
@@ -557,6 +621,14 @@ def write_to_database(
                 )
                 return False
 
+            if not file_has_vehicle_journeys:
+                db_connection.rollback()
```
Comment (alvak1) [inline packages/ref-data-uploaders/txc-uploader/txc_processor.py:622]: Will vehicle journey be available in all the xml files? If yes has this been confirmed?

Comment (ljones92) [reply packages/ref-data-uploaders/txc-uploader/txc_processor.py:622]: Steve's confirmed there should always be vehicle journeys yeah

File: packages/api/client.ts (hunk 1 above)
Comment (alainajaffer) [reply packages/api/client.ts]: as discussed it is causing typescript issues

File: packages/ref-data-uploaders/txc-uploader/txc_processor.py (hunk 2 above)
Comment (alainajaffer) [reply packages/ref-data-uploaders/txc-uploader/txc_processor.py:595]: just following how journey patterns does it

File: packages/ref-data-uploaders/txc-uploader/txc_processor.py (hunk 3 above)
Comment (alainajaffer) [reply packages/ref-data-uploaders/txc-uploader/txc_processor.py:622]: as discussed

File: packages/ref-data-uploaders/txc-uploader/txc_processor.py (hunk 2 above)
Comment (alvak1) [reply packages/ref-data-uploaders/txc-uploader/txc_processor.py:595]: Journey pattern sections depends on the journey pattern id - in this case there are no dependency that would require individual insert

Comment (alainajaffer) [reply packages/ref-data-uploaders/txc-uploader/txc_processor.py:595]: done

File: packages/api/client.ts (hunk 1 above)
Comment (alainajaffer) [reply packages/api/client.ts]: done

File: packages/ref-data-uploaders/txc-uploader/txc_processor.py [hunk 4]
```diff
@@ -252,20 +276,36 @@ def get_admin_area_codes(cursor: aurora_data_api.AuroraDataAPICursor, stop_codes
...
+    cursor: aurora_data_api.AuroraDataAPICursor, vehicle_journeys_info
+):
+    values = [
+       {
+            "vehicle_journey_code": vehicle_journey_info["vehicle_journey_code"],
+            "service_ref": vehicle_journey_info["service_ref"],
+            "line_ref": vehicle_journey_info["line_ref"],
+            "journey_pattern_ref": vehicle_journey_info["journey_pattern_ref"],
+        }
+        for vehicle_journey_info in vehicle_journeys_info
```
Comment (alvak1) [inline packages/ref-data-uploaders/txc-uploader/txc_processor.py:304]: Observed an error "'NoneType' object is not iterable" while running bods function locally

Comment (alainajaffer) [reply packages/ref-data-uploaders/txc-uploader/txc_processor.py:304]: done

**Review by ljones92 — APPROVED** [review]

**Pull request merged**